.travis.yml

```diff
@@ -17,9 +17,11 @@ env:
 
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="database_cleanup"
+  - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="database_cleanup"
   - TESTS="1" ODOO_REPO="OCA/OCB" MAKEPOT="1"
```

Comment:
I see that we are not testing against "odoo/odoo" and this line is also including both together, which makes this to fail. Please put:
```
- TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="database_cleanup" MAKEPOT="1"
- TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="database_cleanup" MAKEPOT="1"
```

The MAKEPOT part is for synchronizing translations with weblate.